challenges/functions.js

```diff
@@ -6,25 +6,29 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+const consume = (frick, frack, cb) => {return cb(frick, frack)}
 
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+const add = (frick, frack) => {return cb(frick, frack)}
+const multiply = (frick, frack) => {return cb(frick, frack)}
```

Comment:
All of these are wrong. You are not calling another callback on them, just defining functions and what result they should yield so You can pass them on to `consume` and call.

The proper way to write is:

```
const add = (frick, frack) => {
  return frick + frack;
};
const multiply = (frick, frack) => {
  return frick * frack;
};
```